### ISSUE TITLE
The main server crashes when using auth stuff

### ISSUE THREAD

**Issue by Fingercomp** [hel-repo/hel#3]
## TL;DR version
That's y'know, one of those very important things. I hope the exception is logged on the server, so @MoonlightOwl would really want to check the logs there out, wouldn't you? :stuck_out_tongue:

### Description
When one uses any auth stuff on the main server, instead of a normal response, it returns a very sad error that says,
```
Internal Server Error

The server encountered an unexpected internal server error

(generated by waitress)
```

What make the issue even stranger, neither running the tests, nor running a local version of the app cause this problem, i.e., it works!

### Possible reasons
No clue, but can be deduced after checking out the logs on the server.

### Possible solutions
No clue, too.

**Comment by Fingercomp** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;o9yXx9aY&#x2F;39-auth"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Сервер крашится при запросе на &#x2F;auth</a>

**Comment by Fingercomp** [comment]
The problem was the Python 3.3 incompatibility: the `bytes.hex()` method first appeared only in Python 3.4. The fix is deployed.
